optimum/onnxruntime/modeling_ort.py

```diff
@@ -1317,6 +1319,91 @@ def forward(
         return ImageClassifierOutput(logits=logits)
 
 
+@add_end_docstrings(ONNX_MODEL_END_DOCSTRING)
+class ORTModelForZeroShotImageClassification(ORTModel):
+    """ONNX Model for zero-shot-image-classification tasks. This class officially supports clip, metaclip-2."""
+
+    @dataclass
+    class MultimodalTextAndImageOutput(ModelOutput):
```

Comment:
what about `ZeroShotImageClassificationOutput` ? and we can also put it in modeling_outputs.py file